### DISCUSSION TITLE
error in HPC3 setup compilation

### DISCUSSION THREAD

**Discussion by Aquios7** [BLAST-WarpX/warpx#4687] [Q&A] [answered]
I was trying to set up warpx on HPC3 at UCI, and I got the following error while going through the setup process on the warpx readthedocs. Have I done something wrong?

[warpx_logfile.txt](https://github.com/ECP-WarpX/WarpX/files/14227270/warpx_logfile.txt)

**Comment by ax3l** [comment, marked as answer]
Thanks for the report and details!

> bash $HOME/src/warpx/Tools/machines/hpc3-uci/install_gpu_dependencies.sh
> ...
> g++: fatal error: Killed signal terminated program cc1plus
> compilation terminated.

That means we used too many resources (cores, in particular) when compiling the dependencies.

Let's use less cores for compiling, it's a bit trial and error how much a system allows in a stable manner: https://github.com/ECP-WarpX/WarpX/pull/4694
